ui/src/Room.tsx

```diff
@@ -241,6 +279,35 @@ export const Room = ({
                             <FullScreenIcon fontSize="large" />
                         </IconButton>
                     </Tooltip>
+
+                    <Tooltip title="Display mode" arrow>
+                        <IconButton
```

Comment:
Thanks for your contribution!

Can we move this setting into the triple dot menu? 

Also, it would be great, if the selected display mode will be saved into the local storage, what do you think?
(I can add this in a separate PR if you don't want to do it (: )

Reply:
Yes, sound good. I'll move the setting into the tripple dot menu and add the persistence using local storage aswell :)